src/lib.rs

```diff
@@ -315,6 +323,9 @@ impl Drop for Executor<'_> {
             }
             drop(active);
 
+            for local_queue in state.local_queue.iter() {
+                while local_queue.queue.pop().is_ok() {}
```

Comment:
Can this be subject to race conditions, or does that even matter?

Reply:
Since it's being dropped there are no active runner queues.